invictus_dashboard/src/app/shared/default-dialog.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, EventEmitter, Inject, Output } from '@angular/core';
+import {
+  MatDialogModule,
+  MatDialogRef,
+  MAT_DIALOG_DATA,
+} from '@angular/material/dialog';
+
+@Component({
+  selector: 'default-dialog',
+  template: `<h1 mat-dialog-title>Warning</h1>
```

Comment:
templateUrl and a different file for HTML is a more standard practice. It makes it easier to read and change. You can still have shared components that use the best practice separation.

Reply:
Gotcha. In this case, I should have deleted this shared file anyway since I decided not to use Angular Materials. I will keep this in mind for the future.